lib/src/config/config.rs

```diff
@@ -249,22 +249,14 @@ impl From<Option<config_file::DatabaseSettings>> for DatabaseConfig {
 pub enum StorageConfig {
     /// This will store the database in a volatile storage. After the client shuts
     /// down all data will be lost.
+    /// This can be used for browser based environments
     ///
     Volatile,
 
     /// This will store the database and key files at specific paths. This is not available when
     /// compiled to WebAssembly.
     ///
     Filesystem(FileStorageConfig),
-
-    /// This will store the database in the browser using *IndexedDB*, and the key files in
-    /// *LocalStorage*. This is only available when run in the browser.
-    ///
-    /// # Notes
-    ///
-    /// There no support for WebAssembly yet.
-    ///
-    Browser,
```

Comment:
Looks like this generated some warnings of `unreachable` code where this `enum` was used for `match`es